test/jdk/java/util/zip/ZipFile/TestZipFileEncodings.java

```diff
@@ -115,6 +116,23 @@ public void testUnicodeManyEntries(String charsetName) throws Throwable {
         test(70000, 10, true, Charset.forName(charsetName));
     }
 
+    @Test
+    public void latin1NotAscii() throws IOException {
+
+        Path zip = Path.of("latin1-not-ascii.zip");
+
+        // latin1, but not ASCII
+        String entryName = "smörgårdsbord";
+
```

Comment:
@cl4es Are we ok with non-ASCII in source files? I'd hate to escape this ;-)

Reply:
As long as the file is UTF-8 encoded then I think it should be fine. Thanks for fixing the spelling before I could remark on it! :D